nimare/meta/mkda.py

```diff
@@ -661,7 +661,7 @@ def correct_fwe_montecarlo(self, result, n_iters=10000, n_cores=-1):
         perm_max_values = perm_results
 
         # Voxel-level FWE
-        vfwe_map = of_values.copy()
+        vfwe_map = of_values.copy().astype(float)
```

Comment:
`numpy.ndarray.astype()` returns a copy by default, so you can drop the old `copy()`.